src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/PosixSignalRegistrationTests.Unix.cs

```diff
@@ -230,7 +210,6 @@ private static void kill(PosixSignal sig)
         }
 
         [DllImport(Interop.Libraries.SystemNative, EntryPoint = "SystemNative_GetPlatformSignalNumber")]
-        [SuppressGCTransition]
```

Comment:
Is there anything actually problematic in this method, or do you just think it is not worth it to use SuppressGCTransition here?

Reply:
This is test code.  Seemed unnecessary to use the attribute here, but to my knowledge there's nothing problematic with doing so.